src/proto/ctl/storage_nvme.proto

```diff
@@ -79,8 +79,9 @@ message NvmeController {
 		uint32 rank = 4;		// DAOS I/O Engine using controller
 		uint64 total_bytes = 5;		// blobstore clusters total bytes
 		uint64 avail_bytes = 6;		// blobstore clusters available bytes
-		string tr_addr = 7;		// transport address of the blobstore
-		string dev_state = 8;		// NVMe device state
+		uint64 cluster_size = 7;        // blobstore cluster size in bytes
```

Comment:
Likewise here.

Reply:
- [ ] Move `cluster_size` field at the end of message payload